core/src/main/java/org/apache/iceberg/SnapshotSummary.java

```diff
@@ -236,6 +248,10 @@ void clear() {
     void addTo(ImmutableMap.Builder<String, String> builder) {
       setIf(addedFiles > 0, builder, ADDED_FILES_PROP, addedFiles);
       setIf(removedFiles > 0, builder, DELETED_FILES_PROP, removedFiles);
+      setIf(addedEqDeleteFiles > 0, builder, ADD_EQ_DELETE_FILES_PROP, addedEqDeleteFiles);
+      setIf(removedEqDeleteFiles > 0, builder, REMOVED_EQ_DELETE_FILES_PROP, removedEqDeleteFiles);
+      setIf(addedPosDeleteFiles > 0, builder, ADD_POS_DELETE_FILES_PROP, addedPosDeleteFiles);
+      setIf(removedPosDeleteFiles > 0, builder, REMOVED_POS_DELETE_FILES_PROP, removedPosDeleteFiles);
       setIf(addedDeleteFiles > 0, builder, ADDED_DELETE_FILES_PROP, addedDeleteFiles);
       setIf(removedDeleteFiles > 0, builder, REMOVED_DELETE_FILES_PROP, removedDeleteFiles);
```

Comment:
[question] since now we are tracking added / removed delete files at a more granular level (i.e pos delete added / removed & eq delete added / removed delete files) these metrics can be re-created using them, should we remove or deprecate them from snapshot summary ?

Reply:
Deprecating things is quite a bit of work for little gain here. I'd just continue to update them.